hazelcast/src/main/java/com/hazelcast/wan/WanReplicationService.java

```diff
@@ -124,6 +124,8 @@
      *                                       {@link com.hazelcast.internal.cluster.Versions#CURRENT_CLUSTER_VERSION}
      * @throws InvalidConfigurationException if there is no WAN replication
      *                                       config for {@code wanReplicationName}
+     * @throws IllegalArgumentException      if {@code fromTimestamp} is greater than
```

Comment:
Same comment as for the EE part: I think we should not allow equality here.

Reply:
Changed the javadoc, please take a look